pucauto.py

```diff
@@ -97,14 +90,11 @@ def load_full_trade_list():
     infinite scroll table. Without this function, we could only see a portion of the cards available for trade."""
 
     old_scroll_y = 0
-    while True:
-        DRIVER.execute_script("window.scrollBy(0, 5000);")
+    new_scroll_y = -1
+    while old_scroll_y != new_scroll_y:
```

Comment:
My inner conditional changed recently to `if new_scroll_y == old_scroll_y or new_scroll_y < old_scroll_y` because of PucaTrade's crappy infinite scroll implementation. Sometimes at the bottom of their page the table grows then shrinks in a weird way that was causing this while loop to run longer than needed because `new_scroll_y` would actually be less than `old_scroll_y` by a few pixels which doesn't make any sense.

I think your while condition will need to change to `while new_scroll_y != old_scroll_y and new_scroll_y > old_scroll_y` and you'll have to initialize `old_scroll_y` to -1 and `new_scroll_y` to 0.